.chloggen/users_sourabhjain_moreoperationmetrics.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: db
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Added `db.client.operation.count` metric to track the number of operations performed by the client.
```

Comment:
do we need `db.client.operation.count`? e.g. we don't (currently) have `http.(client|server).request.count`, under the assumption that you can get count from `http.(client|server).request.duration`